e2e/tests/localAuthoritySubmitsApplication_@slow_test.js

```diff
@@ -231,29 +241,41 @@ Scenario('local authority enters respondents @create-case-with-mandatory-section
+  I.seeInTab(['Respondents 1', 'What is the respondent\'s relationship to the child or children in this case?'], respondents[0].relationshipToChild);
   I.seeInTab(['Respondents 1', 'Do you need contact details hidden from other parties?'], 'No');
   I.seeInTab(['Respondents 1', 'Do you believe this person will have problems with litigation capacity (understanding what\'s happening in the case)?'], 'Yes');
   I.seeInTab(['Respondents 1', 'Give details, including assessment outcomes and referrals to health services'], 'mock reason');
+  I.seeInTab(['Respondents 1', 'Representative', 'Representative\'s first name'], respondents[0].solicitor.firstName);
+  I.seeInTab(['Respondents 1', 'Representative', 'Representative\'s last name'], respondents[0].solicitor.lastName);
+  I.seeInTab(['Respondents 1', 'Representative', 'Email address'], respondents[0].solicitor.email);
 
   I.seeInTab(['Respondents 2', 'Party', 'First name'], respondents[1].firstName);
   I.seeInTab(['Respondents 2', 'Party', 'Last name'], respondents[1].lastName);
   I.seeInTab(['Respondents 2', 'Party', 'Date of birth'], '1 Jan 1955');
   I.seeInTab(['Respondents 2', 'Party', 'Gender'], respondents[1].gender);
-  I.seeInTab(['Respondents 2', 'Party', 'Place of birth'], respondents[1].placeOfBirth);
-  I.seeInTab(['Respondents 2', 'What is the respondent\'s relationship to the child or children in this case?'], 'mock reason');
+  I.seeInTab(['Respondents 2', 'What is the respondent\'s relationship to the child or children in this case?'], respondents[1].relationshipToChild);
   I.seeInTab(['Respondents 2', 'Do you need contact details hidden from other parties?'], 'Yes');
   I.seeInTab(['Respondents 2', 'Give reason'], 'mock reason');
   I.seeInTab(['Respondents 2', 'Do you believe this person will have problems with litigation capacity (understanding what\'s happening in the case)?'], 'No');
 
+  I.seeInTab(['Respondents 3', 'Party', 'First name'], respondents[2].firstName);
+  I.seeInTab(['Respondents 3', 'Party', 'Last name'], respondents[2].lastName);
+  I.seeInTab(['Respondents 3', 'Party', 'Date of birth'], '4 Apr 1978');
+  I.seeInTab(['Respondents 3', 'Party', 'Gender'], respondents[2].gender);
+  I.seeInTab(['Respondents 3', 'What is the respondent\'s relationship to the child or children in this case?'], respondents[2].relationshipToChild);
+  I.seeInTab(['Respondents 3', 'Do you need contact details hidden from other parties?'], 'No');
+  I.seeInTab(['Respondents 3', 'Do you believe this person will have problems with litigation capacity (understanding what\'s happening in the case)?'], 'No');
+  I.seeInTab(['Respondents 3', 'Representative', 'Representative\'s first name'], respondents[2].solicitor.firstName);
+  I.seeInTab(['Respondents 3', 'Representative', 'Representative\'s last name'], respondents[2].solicitor.lastName);
+  I.seeInTab(['Respondents 3', 'Representative', 'Email address'], respondents[2].solicitor.email);
```

Comment:
@joez-f would it be worth checking the additional unregistered details here too, something like (untested):
```suggestion
  I.seeInTab(['Respondents 3', 'Representative', 'Email address'], respondents[2].solicitor.email);
  I.seeInTab(['Respondents 3', 'Representative', 'Organisation (unregistered)', 'Organisation name'], respondents[2].solicitor.unregisteredOrganisation.name);
  I.seeInTab(['Respondents 3', 'Representative', 'Organisation (unregistered)', 'Address', 'Building and Street'], respondents[2].solicitor.unregisteredOrganisation.address.buildingAndStreet.lineOne);
  I.seeInTab(['Respondents 3', 'Representative', 'Organisation (unregistered)', 'Address', 'Address Line 2'], respondents[2].solicitor.unregisteredOrganisation.address.buildingAndStreet.lineTwo);
  I.seeInTab(['Respondents 3', 'Representative', 'Organisation (unregistered)', 'Address', 'Address Line 3'], respondents[2].solicitor.unregisteredOrganisation.address.buildingAndStreet.lineThree);
  I.seeInTab(['Respondents 3', 'Representative', 'Organisation (unregistered)', 'Address', 'Town or City'], respondents[2].solicitor.unregisteredOrganisation.address.town);
  I.seeInTab(['Respondents 3', 'Representative', 'Organisation (unregistered)', 'Address', 'Postcode/Zipcode'], respondents[2].solicitor.unregisteredOrganisation.address.postcode);
  I.seeInTab(['Respondents 3', 'Representative', 'Organisation (unregistered)', 'Address', 'Country'], respondents[2].solicitor.unregisteredOrganisation.address.country);
```

Reply:
These exact lines are included in a followup PR, so no point adding here.

Reply:
will add to follow up PR, currently it is missing, follow up only checks managing org not unregistered org